<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.core.i18n.NoSuchMessageExceptionSpec" tests="2" skipped="0" failures="0" errors="0" timestamp="2017-03-29T07:30:24" hostname="testing-gce-66c09752-be5b-4dc1-8b4c-7bf7aedc91bc" time="0.006">
  <properties/>
  <testcase name="Exercise constructor with errors" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.005"/>
  <testcase name="Successfully create exception" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
